core/shared/src/main/scala/matryoshka/patterns/PotentialFailure.scala

```diff
@@ -34,9 +34,7 @@ final case class PartialFailure[T[_[_]], F[_], E, A] private[patterns](v: F[A])
     extends PotentialFailure[T, F, E, A]
 
 object PotentialFailure {
-  implicit def potentialFailureEqual[T[_[_]], F[_], E: Equal](
-    implicit T: Equal[T[F]], F: Delay[Equal, F]):
-      Delay[Equal, PotentialFailure[T, F, E, ?]] =
+  implicit def potentialFailureEqual[T[_[_]], F[_], E: Equal](implicit T: Equal[T[F]], F: Delay[Equal, F]): Delay[Equal, PotentialFailure[T, F, E, ?]] =
```

Comment:
Basically, it’s a wart that is currently (has to be?) applied to the source code, and the current regex(?) isn’t smart enough to notice the pattern broken across lines – wartremover/wartremover#226